apps/graph-api/src/api.config.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('Account API Config', () => {
     API_BODY_JSON_LIMIT: undefined,
     API_PORT: undefined,
     API_TIMEOUT_MS: undefined,
-    FREQUENCY_HTTP_URL: undefined,
+    SIWF_NODE_RPC_URL: undefined,
     SIWF_URL: undefined,
```

Comment:
nit: We already have `SIWF_URL`. Is this redundant or they are used differently?

Reply:
They are different.

Reply:
Do we need to also change SIWF_URL to be SIWF_UI_URL? @mattheworris @aramikm

Reply:
I do like `SIWF_UI_URL` but I'm fine with leaving `SIWF_URL` too, as you prefer.